components/Mappings/NewFieldModal.js

```diff
@@ -117,23 +123,20 @@ export default class NewFieldModal extends Component {
 						<span className="col">
 							Data Type
 						</span>
+
 					</Header>
 					<div style={{ padding: '10px 0', display: 'flex' }}>
 						<span style={{ width: 150, marginRight: 12 }}>
-							<Creatable
-								value={{ label: this.state.esType, value: this.state.esType }}
-								placeholder="Select or Create Type"
-								promptTextCreator={label => `Create type "${label}"`}
-								onChange={this.handleEsTypeChange}
-								options={
-									this.props.types
-										.map(item => ({
-											value: item,
-											label: item,
-										}))
-								}
-								clearable={false}
-							/>
+							<Dropdown overlay={menu}>
+								<Input
+									innerRef={(el) => { this.input = el; }}
```

Comment:
Where are we using `this.input`?

Also, it would be great if we can use the new `ref` API instead? Refer: https://reactjs.org/docs/refs-and-the-dom.html#creating-refs